biz.aQute.bndlib/src/aQute/bnd/cdi/CDIAnnotations.java

```diff
@@ -115,7 +115,7 @@ public boolean analyzeJar(Analyzer analyzer) throws Exception {
 		for (Clazz c : analyzer.getClassspace()
 			.values()) {
 
-			if (c.isModule() || c.isEnum() || c.isInterface() || c.isInnerClass() || c.isSynthetic()
+			if (c.isModule() || c.isEnum() || c.isInnerClass() || c.isSynthetic()
```

Comment:
This also means you will process annotations which are interfaces. Is that intended?

Reply:
hmm, good question... I suspect no.